docs/installation/database/passing-configuration.rst

```diff
@@ -0,0 +1,430 @@
+
+.. py:currentmodule:: datacube
+
+ODC Configuration (Details)
+***************************
+
+Open Data Cube configuration files, supported configuration options, and basic use cases and defaults
+are described in :doc:`configuration`.
+
+The Open Data Cube uses configuration files and/or environment variables to determine how to connect to databases.
```

Comment:
Can there be connections to multiple databases at the same time? If not, it might be more clear to talk about the/a database.

Reply:
Yes there can!  Several of the changes to the configuration layer in 1.9 are driven by the need for better support for multi-database scenarios.